### ISSUE TITLE
Insufficient contrast in navbar

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4857]
Navbar text does not pass the contrast criteria for small text. The minimum contrast level is 4.5:1 while this text is at 2:1. The contrast is also not high enough if you hover over it. So, both light grey and white-ish have a to low contrast level:

<img width="271" height="33" alt="Image" src="https://github.com/user-attachments/assets/987f269c-89ab-428f-bc7f-0185882ed9e3" />
